jerry-core/parser/js/js-parser-module.c

```diff
@@ -413,10 +420,8 @@ parser_module_parse_import_clause (parser_context_t *context_p) /**< parser cont
 
     lexer_construct_literal_object (context_p, &context_p->token.lit_location, LEXER_IDENT_LITERAL);
 
-    local_name_p = ecma_new_ecma_string_from_utf8 (context_p->lit_object.literal_p->u.char_p,
-                                                   context_p->lit_object.literal_p->prop.length);
-    ecma_ref_ecma_string (local_name_p);
-    import_name_p = local_name_p;
+    uint16_t local_name_index = context_p->lit_object.index;
+    uint16_t import_name_index = PARSER_MAXIMUM_NUMBER_OF_LITERALS;
```

Comment:
The naming is mixed up here a bit, it should be the opposite of the export case. We have an import name first, and an optional local name later. Please switch these up.